From: Andrew Morton <akpm@linux-foundation.org>
Subject: zram-fixup-read_block_state-fix-2
Date: Thu Jan 15 10:30:48 PM PST 2026

fix build

drivers/block/zram/zram_drv.c: In function 'read_block_state':
drivers/block/zram/zram_drv.c:1582:41: error: format '%lu' expects argument of type 'long unsigned int', but argument 6 has type 'int' [-Werror=format=]
 1582 |                         "%12zd %12u.%06lu %c%c%c%c%c%c\n",
      |                                     ~~~~^
      |                                         |
      |                                         long unsigned int
      |                                     %06u
 1583 |                         index, zram->table[index].attr.ac_time, 0,
      |                                                                 ~
      |                                                                 |
      |                                                                 int

Cc: Sergey Senozhatsky <senozhatsky@chromium.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 drivers/block/zram/zram_drv.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/block/zram/zram_drv.c~zram-fixup-read_block_state-fix-2
+++ a/drivers/block/zram/zram_drv.c
@@ -1589,7 +1589,7 @@ static ssize_t read_block_state(struct f
 			goto next;
 
 		copied = snprintf(kbuf + written, count,
-			"%12zd %12u.%06lu %c%c%c%c%c%c\n",
+			"%12zd %12u.%06d %c%c%c%c%c%c\n",
 			index, zram->table[index].attr.ac_time, 0,
 			test_slot_flag(zram, index, ZRAM_SAME) ? 's' : '.',
 			test_slot_flag(zram, index, ZRAM_WB) ? 'w' : '.',
_
